### PR TITLE
nso_verify handle leaf-list in 4.5 and identityref (#37393)

### PR DESCRIPTION
**Pull request by gundalow**
##### SUMMARY
NSO verify did not handle leaf-list value verification in 4.5 and
later due to changes made for configuration writing made.

map prefix for identityref types in verification.
(cherry picked from commit 6308047dc9bbc72a180cfad1967ef95f6090991e)

https://github.com/ansible/ansible/pull/37393

### CODE REVIEW

File: test/units/modules/network/nso/nso_module.py [hunk 1]
```diff
@@ -60,7 +60,11 @@ def read(self):
         return self.body
 
 
-def mock_call(calls, url, data=None, headers=None, method=None):
+def mock_call(calls, url, timeout, data=None, headers=None, method=None):
```
Comment (cnasten) [inline test/units/modules/network/nso/nso_module.py]: timeout, needs to be removed here.

**Comment by mattclay** [comment]
CI failure in unit tests: https://app.shippable.com/github/ansible/ansible/runs/58406/3/tests

File: test/units/modules/network/nso/nso_module.py (hunk 1 above)
Comment (gundalow) [reply test/units/modules/network/nso/nso_module.py]: Removed, waiting on Shippable

**Comment by gundalow** [comment]
Merged into `stable-2.5`, will be released in `2.5.1`

**Pull request merged**